controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const endpointsJson = require("../endpoints.json");
+
+const getTopics = (req, res) => {
+    res.status(200).send ({topics: endpointsJson})
```

Comment:
We shouldn't be getting the topics from our endpoints.json file - we should make a query to our Database and get our topics from there!